assets/xml/objects/object_daiku.xml

```diff
@@ -66,11 +66,11 @@
         <Limb Name="object_daiku_Standardlimb_00A7F8" Type="Standard" EnumName="OBJECT_DAIKU_LIMB_0F" Offset="0xA7F8" />
         <Limb Name="object_daiku_Standardlimb_00A804" Type="Standard" EnumName="OBJECT_DAIKU_LIMB_10" Offset="0xA804" />
         <Skeleton Name="object_daiku_Skel_00A850" Type="Flex" LimbType="Standard" LimbNone="OBJECT_DAIKU_LIMB_NONE" LimbMax="OBJECT_DAIKU_LIMB_MAX" EnumName="ObjectDaikuLimb" Offset="0xA850" />
-        <Animation Name="object_daiku_Anim_00ACD0" Offset="0xACD0" />
-        <Animation Name="object_daiku_Anim_00B690" Offset="0xB690" />
-        <Animation Name="object_daiku_Anim_00BEAC" Offset="0xBEAC" />
-        <Animation Name="object_daiku_Anim_00C234" Offset="0xC234" />
-        <Animation Name="object_daiku_Anim_00C92C" Offset="0xC92C" />
-        <Animation Name="object_daiku_Anim_00D328" Offset="0xD328" />
+        <Animation Name="object_daiku_Anim_00ACD0" Offset="0xACD0" /> <!-- Original name is "dk2_roya" -->
+        <Animation Name="object_daiku_Anim_00B690" Offset="0xB690" /> <!-- Original name is "dk2_shiji" (perhaps "instructions; directions​"?) -->
+        <Animation Name="object_daiku_Anim_00BEAC" Offset="0xBEAC" /> <!-- Original name is "dk2_shiji02" -->
+        <Animation Name="object_daiku_Anim_00C234" Offset="0xC234" /> <!-- Original name is "dk2_suwari" ("sitting") -->
+        <Animation Name="object_daiku_Anim_00C92C" Offset="0xC92C" /> <!-- Original name is "dk2_tasuke" ("to save; to rescue​") -->
```

Comment:
Oh neat this animation survived from OOT, huh.

Reply:
Looking at EnDaiku, it looks like there's code to play this animation? I have no idea under what conditions it is triggered though.

Reply:
Uhhh... in hindsight maybe the credits shows them sitting down?